### ISSUE TITLE
Add space between "Category groups" and "User groups"

### ISSUE THREAD

**Issue by lindseydiloreto** [lindseydiloreto/DashCols-Craft#3]
On the DashCols home page, there isn't much breathing room between "Category groups" and "User groups"...
![crowded](https://cloud.githubusercontent.com/assets/5309692/10275504/d6dee200-6afc-11e5-9573-1ad0d413f53a.png)

**Comment by mmikkel** [comment]
Right. Slight CSS mediocrity there.

I'm also thinking "All Users" might be more appropriate under "Listings", as it isn't a user group per se.

Actually, when I think about it, it might be even better to just list everything by element type. Something like...

*Entries*
* All entries
* Singles
* Section A
* Section B

*Categories*
* Category group A
* Category group B

*Users*
* All users
* User group A
* User group B

Seems a bit more logical, no?

**Comment by lindseydiloreto** [comment]
Yeah, that actually does make a lot more sense! Grouping by element type is the way to go. :)
